DiceKit/FrequencyDistribution.swift

```diff
@@ -216,4 +216,12 @@ extension FrequencyDistribution {
         }
     }
     
+    public func minOutcome() -> Int {
+        return self.orderedOutcomes.first ?? 0
```

Comment:
It would only be 0 when it's empty. And since you can't find that outcome there, no. It should return `nil`.